src/System.Security.Cryptography.Pkcs/pkg/System.Security.Cryptography.Pkcs.builds

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Security.Cryptography.Pkcs.pkgproj" />
```

Comment:
I don't know how to answer that question :).  I don't think we'd want it for netcli's shared framework, just leaving it as nuget-referenced package (or private shared framework redist).